apis/tags/v1beta1/tagbinding_parent.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/apis/common/identity"
+	refsv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	runv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/run/v1beta1"
+	storagev1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/storage/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ProjectPrefix = "//cloudresourcemanager.googleapis.com/projects"
+	// OrganizationPrefix = "//cloudresourcemanager.googleapis.com/organizations"
+	BucketPrefix = "//storage.googleapis.com/projects/_/buckets"
+	RunJobPrefix = "//run.googleapis.com"
+)
+
+var _ identity.Identity = &TagBindingParent{}
```

Comment:
I wonder if we should work towards adding a `ServiceName()` to identity.Identity